packages/resources/src/RemixSite.ts

```diff
@@ -789,9 +798,13 @@ export class RemixSite extends Construct implements SSTConstruct {
       platform: "node",
       external: ["aws-sdk"],
       outfile: path.join(outputPath, "server.js"),
-      // Need to add the --ignore-annotations flag to ESBuild. It appears Remix templates by default include the "sideEffects": false, flag in their package.json. This causes ESBuild to remove the required polyfill module. I had removed the flag so it was working fine for me, but likely we will want to avoid having everyone update their Remix app like this. Remix includes polyfills so it is always going to be by nature with side effects. I've asked on their GH the reasoning for the flag being there in the first place. There is also an active PR to remove the need for this flag in their package.json, although the PR doesn't provide the original motivation for the flag being there. I will do more research and get back to ya'll. For sure we will want to address this. https://esbuild.github.io/api/#ignore-annotations
-      ignoreAnnotations: true,
-    })
+      // We need to ensure that the polyfills are injected above other code that
+      // will depend on them. Importing them within the top of the lambda code
+      // doesn't appear to guarantee this, we therefore leverage ESBUild's
+      // `inject` option to ensure that the polyfills are injected at the top of
+      // the bundle.
+      inject: [polyfillDest],
```

Comment:
Does passing in `polyfillSource` not work?

Reply:
Unfortunately not. That was my original implementation, but then ESBuild tried to do package resolution to the referenced remix package which failed your unit tests. If we added the remix package as a dependency to your resources package, or higher then it could work. As a dev dependency maybe?